app/src/main/java/sq/rogue/rosettadrone/DroneModel.java

```diff
@@ -822,6 +835,11 @@ void send_autopilot_version() {
         msg_autopilot_version msg = new msg_autopilot_version();
         msg.capabilities = MAV_PROTOCOL_CAPABILITY.MAV_PROTOCOL_CAPABILITY_COMMAND_INT;
         msg.capabilities |= MAV_PROTOCOL_CAPABILITY.MAV_PROTOCOL_CAPABILITY_MISSION_INT;
+        msg.os_sw_version = 0x040107;
+        msg.middleware_sw_version = 0x040107;
+        msg.flight_sw_version = 0x040107;
```

Comment:
@The1only Just wanted to let you know that this silences the startup fw version error that occurs with QGroundControl (tho I dont think version is encoded right, but it does the trick..)
Fwiw I am playing around with uGCS too and there it still chokes on receiving version info's until a timeout occurs, so it might be waiting for *something* else in this msg.